### PR TITLE
docs: support PWA

### LINKED ISSUE #110: PWA docs

### PR DESCRIPTION
**Pull request by hydrati**
### Description

Add ~~**partital**~~ PWA support ~~(only service worker caching)~~ for the docs website.

It's not necessary to add full PWA support for a docs website, I think.

In addition, you may need to provide bitmap icons (like `icon_256x256.png`...) for this application.

### Linked Issues
closes #110

### CODE REVIEW

**Review by sxzz — APPROVED** [review]

**Comment by sxzz** [comment]
There's a problem on https://unplugin-vue-macros-f0fwu9z6n-sxzz.vercel.app/macros/define-render. And then refresh the page.
image

**Comment by hydrati** [comment]
> There's a problem on https://unplugin-vue-macros-f0fwu9z6n-sxzz.vercel.app/macros/define-render. And then refresh the page.
> 
> image

Ok, Fixed.

File: docs/.vitepress/config.ts [hunk 1]
```diff
@@ -1,63 +1,135 @@
...
+      reactivityTransform: true,
+    },
 
-  themeConfig: {
-    logo: '/favicon.svg',
-    footer: {
-      message: 'Made with ❤️',
-      copyright: 'MIT License © 2022 三咲智子',
+    themeConfig: {
+      logo: '/logo.svg',
```
Comment (sxzz) [inline docs/.vitepress/config.ts:52]: Same here

File: docs/vite.config.ts [hunk 2]
```diff
@@ -4,5 +4,11 @@ import DefineOptions from 'unplugin-vue-define-options/vite'
 import Jsx from '@vitejs/plugin-vue-jsx'
 
 export default defineConfig({
+  build: {
+    // sourcemap: true,
+    // minify: false,
```
Comment (sxzz) [inline docs/vite.config.ts:9]: Remove them

**Review by sxzz — APPROVED** [review]

**Pull request merged**